graphite/graphite.go

```diff
@@ -33,10 +33,13 @@ const (
 	Version = 5
```

Comment:
Concern incrementing the version of this plugin, so snap-plugin-publisher-graphite in ver 6 (with fixed the bug) might be released.